kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/WatchConnectionManager.java

```diff
@@ -308,7 +308,7 @@ public void waitUntilReady() {
   @Override
   public void close() {
     logger.debug("Force closing the watch {}", this);
-    forceClosed.set(true);
+    closeEvent(null);
```

Comment:
this `null` looks like it could make it through to a parameter of the `watcher.onClose` callback.  Is the contract that the cause can be nullable?  I'm guessing not all custom watches check for null in their `onClose` methods